ci/helm-chart/README.md

```diff
@@ -10,8 +10,8 @@ This chart is community maintained by [@Matthew-Beckett](https://github.com/Matt
 ## TL;DR;
 
 ```console
-$ git clone https://github.com/cdr/code-server.git
-$ helm install code-server/charts/code-server
+$ git clone https://github.com/cdr/code-server
+$ helm install code-server code-server/ci/helm-chart
```

Comment:
what about just `/ci/helm` the `chart` suffix seems implicit

Reply:
For me at least it helps to be clear that the directory itself is the chart. It's not like a bunch of scripts to generate the chart or anything.

Reply:
I agree, in most other repositories that maintain charts they are usual in a charts directory at the root, it's unusual for it to be called CI, it might be confusing since it's not build scripts or CI related wizardry.

Reply:
> I agree, in most other repositories that maintain charts they are usual in a charts directory at the root, it's unusual for it to be called CI, it might be confusing since it's not build scripts or CI related wizardry.
 
At some point we will want to automate publishing the helm chart so it'll at least be somewhat used in CI.

We tend to keep all our release related stuff in the `ci` directory to prevent root pollution. In terms of discovery, I think the link I added in `install.md` should be enough.